### PR TITLE
added chainermn.testing.to_device function

### PR DESCRIPTION
**Pull request by keisukefukuda**
This PR adds a new ChainerMN subpackage namely`chainermn.testing`.

## Background:
There are ongoing efforts to integrate ChainerMN with ChainerX. To make the process more efficient, we will need some utility functions. For now, only one function `get_device` is added to the new package.

## Changes
Add a function `chainermn.testing.get_device()`.

### CODE REVIEW

File: chainermn/testing/device.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        if use_chx:
+            device = 'cuda:{}'.format(communicator.intra_rank)
+        else:
+            # cupy
+            device = '@cupy:{}'.format(communicator.intra_rank)
+    else:
+        if use_chx:
+            device = 'native:0'
+        else:
+            device = -1
```
Comment (toslunar) [inline chainermn/testing/device.py]:
```suggestion
            device = '@numpy'
```

Comment (keisukefukuda) [reply chainermn/testing/device.py]: thanks!

**Comment by keisukefukuda** [comment]
pfnci, test this please

**Comment by belldandyxtq** [comment]
LGTM, What do you think @shu65

**Pull request merged**